iOS/DuckDuckGo/MainViewController.swift

```diff
@@ -797,9 +797,11 @@ class MainViewController: UIViewController {
 
             viewCoordinator.toolbarTabSwitcherButton.customView = button
         } else {
+            assert(tabSwitcherButton == nil)
             tabSwitcherButton = TabSwitcherButton()
-            tabSwitcherButton.delegate = self
+            tabSwitcherButton?.delegate = self
             viewCoordinator.toolbarTabSwitcherButton.customView = tabSwitcherButton
+            assert(tabSwitcherButton != nil)
```

Comment:
Maybe a little OTT but if for some reason the button isn't being set here hopefully we see that in dev.